pkg/master/thirdparty/thirdparty.go

```diff
@@ -284,6 +284,7 @@ func (m *ThirdPartyResourceServer) InstallThirdPartyResource(rsrc *extensions.Th
 	m.genericAPIServer.HandlerContainer.Add(genericapi.NewGroupWebService(api.Codecs, path, apiGroup))
 
 	m.addThirdPartyResourceStorage(path, plural.Resource, thirdparty.Storage[plural.Resource].(*thirdpartyresourcedataetcd.REST), apiGroup)
+	registered.AddThirdPartyAPIGroupVersions(schema.GroupVersion{Group: group, Version: rsrc.Versions[0].Name})
```

Comment:
Is anything broken in the apiserver due to not registering the thirdpartyresources? We are either missing a test or the whole registration is not necessary. In the later case, I would rather remove the whole registration logic for tprs.